docs/serving/samples/multi-container/README.md

```diff
@@ -0,0 +1,250 @@
+A simple web app written in Go that you can use for multi container testing.
+
+Follow the steps below to create the sample code and then deploy the app to your
+cluster. You can also download a working copy of the sample, by running the
+following command:
+
+```shell
+git clone -b "{{< branch >}}" https://github.com/knative/docs knative-docs
+```
+
+## Prerequisites
+
+- A Kubernetes cluster with Knative installed and DNS configured. Follow the
+  [installation instructions](../../../install/README.md) if you need to
+  create one.
+- [Docker](https://www.docker.com) installed and running on your local machine,
+  and a Docker Hub account configured (we'll use it for a container registry).
+- Make sure multi-container flag is enabled as part of `config-features` configmap.
```

Comment:
I'd move this up to line 2-3

Reply:
Also, Go tool should be part of prerequisites maybe?

Reply:
1. Moved to 2-3

>Also, Go tool should be part of prerequisites maybe?

I don't see mentioning Go tool  as part of pre-requisites in any of the samples